### ISSUE TITLE
HydroSight Outlier Removal

### ISSUE THREAD

**Issue by xinyangf1** [peterson-tim-j/HydroSight#47]
Hi Tim,

I'm trying to use HydroSight Outlier Removal function to remove the outliers in my GW heads file.
However, the system prompts me that I've met a Data Analysis Algorighm failure.
I've attached the error screen shot, the model and the GW head file for your consideration.
Please don't hesitate to let me know if you need any further info.
Thank you so much for your help!

Cheers,
Xinyang

![H3max_hydrograph](https://user-images.githubusercontent.com/51348265/68918786-c1d1f100-07c3-11ea-8891-6fc6a0206077.png)
[HydroSight.zip](https://github.com/peterson-tim-j/HydroSight/files/3849903/HydroSight.zip)

**Comment by peterson-tim-j** [comment]
Thanks. This bore is quite challenging. In the late 1990s there is a large step change. The observations within this period may be true, but simply incorrectly displaced. HydroSight cannot automatically shift this sequence points back. However, after some tweaking with the max. head change per day (and turning off setting for which there was no data, eg casing length), I was able to achieve a good result. Attached is the .mat file. if you are happy with this outcome, could you please close this issue?

[Remove_outlier_test1.zip](https://github.com/peterson-tim-j/HydroSight/files/3894379/Remove_outlier_test1.zip)

**Comment by xinyangf1** [comment]
Hi Tim,

Thank you so much for helping me solve this issue!
I've run your attached model and the outliers have been nicely removed!
The outlier removal function also works for me now. I've changed to use MATLAB R2017b and downloaded all the recommended Toolbox when I install MATLAB R2017b. This is the solution I found to make the Outlier removal function working in my laptop. Just for your info.
Thank you so much for your help! I'm happy to close this issue.

Cheers,
Xinyang
